### PR TITLE
Changed warning box styling

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Warning box with some fresh styling.

Desktop:
Screenshot 2021-01-20 at 11 39 18

Smaller screens:
Screenshot 2021-01-20 at 11 39 36

### CODE REVIEW

File: packages/app/src/components-styled/message-tile.tsx [hunk 1]
```diff
@@ -3,45 +3,70 @@ import styled from 'styled-components';
...
+    fill: 'black',
+  })
+);
+
+const StyledMessageBox = styled(Box)(
+  css({
+    display: 'flex',
+    alignItems: 'center',
+    flexGrow: 1,
+    py: 2 * 1.5,
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/components-styled/message-tile.tsx:62]:
```suggestion
    py: 3,
```

File: packages/app/src/components-styled/message-tile.tsx [hunk 2]
```diff
@@ -3,45 +3,70 @@ import styled from 'styled-components';
...
+    mx: asResponsiveArray({ _: '7px', sm: '21px' }),
+    fill: 'black',
+  })
+);
+
+const StyledMessageBox = styled(Box)(
+  css({
+    display: 'flex',
+    alignItems: 'center',
+    flexGrow: 1,
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/components-styled/message-tile.tsx:61]: For IE iet's better to alway define all properties of `flex`

File: packages/app/src/components-styled/message-tile.tsx [hunk 3]
```diff
@@ -3,45 +3,70 @@ import styled from 'styled-components';
...
+    padding: 0,
+  })
+);
+
+const StyledWarningBox = styled(Box)(
+  css({
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexShrink: 0,
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/components-styled/message-tile.tsx:40]: For IE iet's better to alway define all properties of `flex`

File: packages/app/src/components-styled/message-tile.tsx [hunk 4]
```diff
@@ -3,45 +3,70 @@ import styled from 'styled-components';
...
 }
 
+const StyledTile = styled(Tile)(
+  css({
+    flexDirection: 'row',
+    padding: 0,
+  })
+);
+
+const StyledWarningBox = styled(Box)(
```
Comment (VWSCoronaDashboard) [inline packages/app/src/components-styled/message-tile.tsx:35]: Can't you simply name this `WarningBox`?

File: packages/app/src/components-styled/message-tile.tsx [hunk 5]
```diff
@@ -3,45 +3,70 @@ import styled from 'styled-components';
...
           <Children>{message}</Children>
         )}
-      </Box>
-    </Tile>
+      </StyledMessageBox>
+    </StyledTile>
   );
 }
 
+const StyledTile = styled(Tile)(
```
Comment (VWSCoronaDashboard) [inline packages/app/src/components-styled/message-tile.tsx:28]: WarningTile?

File: packages/app/src/components-styled/message-tile.tsx [hunk 6]
```diff
@@ -3,45 +3,70 @@ import styled from 'styled-components';
...
-    fill: 'black',
     display: 'block',
+    width: asResponsiveArray({ _: 30, sm: 38 }),
+    height: asResponsiveArray({ _: 30, sm: 38 }),
+    mx: asResponsiveArray({ _: '7px', sm: '21px' }),
+    fill: 'black',
+  })
+);
+
+const StyledMessageBox = styled(Box)(
```
Comment (VWSCoronaDashboard) [inline packages/app/src/components-styled/message-tile.tsx:57]: WarningMessageBox?

**Review by VWSCoronaDashboard — COMMENTED** [review]
Nits about naming things

File: packages/app/src/components-styled/message-tile.tsx [hunk 7]
```diff
@@ -3,45 +3,70 @@ import styled from 'styled-components';
...
-    >
-      <Box display="flex" alignItems={['flex-start', 'center']}>
-        <Box mr={3} width={38} flexShrink={0}>
-          <StyledWarning />
-        </Box>
+    <StyledTile>
+      <StyledWarningBox>
+        <StyledWarning />
+      </StyledWarningBox>
+      <StyledMessageBox>
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/components-styled/message-tile.tsx:17]: I'm a little confused here but it's not your fault. The component is called MessageTile, yet it seems to be about a very specific type of message; a warning. So I think this component should get a different name. I suggest you call it WarningMessage and have WarningTile be its first child like @VWSCoronaDashboard proposed.

File: packages/app/src/components-styled/message-tile.tsx (hunk 2 above)
Comment (VWSCoronaDashboard8) [reply packages/app/src/components-styled/message-tile.tsx:61]: @VWSCoronaDashboard4 There is no auto prefixer as part of our style output chain?

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: yarn.lock [hunk 8]
```diff
@@ -5373,6 +5373,11 @@ core-js-pure@^3.0.0:
   resolved "https://registry.yarnpkg.com/core-js-pure/-/core-js-pure-3.8.2.tgz#286f885c0dac1cdcd6d78397392abc25ddeca225"
```
Comment (VWSCoronaDashboard8) [inline yarn.lock:5373]: These lock file changes do not seem to be related to this PR.

File: packages/app/src/components-styled/warning-message.tsx [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+import styled from 'styled-components';
+import Warning from '~/assets/warning.svg';
+import { Box } from './base';
+import { Tile } from '~/components-styled/tile';
+import { asResponsiveArray } from '~/style/utils';
+interface WarningMessageProps {
+  message: React.ReactNode;
+}
+
+export function WarningMessage({ message }: WarningMessageProps) {
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/components-styled/warning-message.tsx:11]: Since the outer component is a Tile, I suggest to rename this component to `WarningTile`. This way we know by reading the name that we can treat it as any other Tile.

File: packages/app/src/components-styled/warning-message.tsx [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+  css({
+    fontSize: 3,
+    fontWeight: 'bold',
+    '> *': {
+      mt: 0,
+      mb: 3,
+      ':last-child': {
+        mb: 0,
+      },
+    },
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/components-styled/warning-message.tsx:78]: I think these styles can be applied using the Box component:
```diff
- <Children>{message}</Children>
+ <Box spacing={3} fontSize={3} fontWeight='bold'>{message}</Box>
```

File: yarn.lock (hunk 8 above)
Comment (VWSCoronaDashboard10) [reply yarn.lock:5373]: Yeah I agree. Can you rollback these changes, merge develop in this branch and re-run yarn in order to see if it helps?
